pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -423,8 +441,7 @@ def process_data(self):
         for qbn in self.data_to_fit:
             if isinstance(self.data_to_fit[qbn], (list, tuple)):
                 self.data_to_fit[qbn] = self.data_to_fit[qbn][0]
-        global_PCA = self.get_param_value('global_PCA', default_value=False)
```

Comment:
if this flag does not work any more, it needs to be removed from the docstring and the code in other modules needs to be refactored

Reply:
It was removed from the docstring of MultiQubit_TimeDomain_Analysis (it appears there because it can be set as the rotation_type parameter). 
I just did a search of the whole timedomain_analysis.py module and I couldn't find the variable global_PCA anywhere. Did you find an exaplicit bug somewhere?

Reply:
Done now :)